amgut/lib/data_access/test/test_survey.py

```diff
@@ -230,6 +229,56 @@ def test_american_name(self):
 
 
 class TestSurvey(TestCase):
+    def insert_data(self):
+        """ Prepare tests by inserting some data into the DB.
+
+        Returns
+        -------
+        survey object, survey_id, notes_test and consent of the newly created
+        survey.
+        """
+
+        # Create random string to test update happens
+        c = ascii_letters + '1234567890'
+        notes_test = ''.join([choice(c) for i in range(40)])
```

Comment:
Thanks for opening the issue - can you also add a TODO in this code to make sure that when we solve #646 we add a non-ascii character in here?